src/main/java/com/google/devtools/common/options/ParamsFilePreProcessor.java

```diff
@@ -49,18 +51,23 @@ public abstract class ParamsFilePreProcessor implements ArgsPreProcessor {
    * @throws OptionsParsingException if the path does not exist.
    */
   @Override
-  public List<String> preProcess(List<String> args) throws OptionsParsingException {
-    if (!args.isEmpty() && args.get(0).startsWith("@")) {
+  public List<ArgAndFallbackData> preProcess(List<ArgAndFallbackData> args)
+      throws OptionsParsingException {
+    if (!args.isEmpty() && args.getFirst().arg.startsWith("@")) {
```

Comment:
I don't think you can use `getFirst()` in this file given the java 11 thing.

Reply:
This actually doesn't break CI since the bootclasspath is determined by the Java runtime version, not the language level. But it would break the consumers that legitimately build with a JDK 11, so good catch on your part.

@cushon @hvadehra Since this is coming up in Bazel code, what do you think of reinvigorating https://github.com/bazelbuild/rules_java/pull/182?

Reply:
I won't take credit, a breakage popped up when I imported.